### PR TITLE
Updates Docs about Svelte Setup & Template

### PR DESCRIPTION
**Pull request by TobiObeck**
This PR fixes documentation issues regarding Svelte project setup and templates that I recently chatted about with David in a [Discord Thread](https://discord.com/channels/795785288994652170/1087844443933118596/1087882795365441567).

This PR should be considered in conjunction with a PR ("Updates Svelte Template link") in the new docs repo (linking to the template introduced in this PR).
https://github.com/statelyai/docs/pull/74

### CODE REVIEW

File: docs/recipes/svelte.md [hunk 1]
```diff
@@ -1,8 +1,15 @@
 # Usage with Svelte
 
-XState integrates great with Svelte, and especially [Svelte stores!](https://svelte.dev/docs#svelte_store)
+In this section you find information for setting up XState in a Svelte project. Nowadays the Svelte docs recommend the usage of Vite for new Svelte or SvelteKit projects. The section directly below is intended for such projects. If you want to integrate XState into older projects that rely on rollup, conduct the section further below.
```
Comment (davidkpiano) [inline docs/recipes/svelte.md:3]:
```suggestion
In this section, we provide guidance on setting up XState in a Svelte project. For new Svelte or SvelteKit projects, we recommend using [Vite](https://vitejs.dev/guide/) as your build tool. This documentation is tailored to such projects. If you're working on an older project that relies on Rollup, please refer to the section further below.
```

File: docs/recipes/svelte.md [hunk 2]
```diff
@@ -1,8 +1,15 @@
 # Usage with Svelte
 
-XState integrates great with Svelte, and especially [Svelte stores!](https://svelte.dev/docs#svelte_store)
+In this section, we provide guidance on setting up XState in a Svelte project. For new Svelte or SvelteKit projects, we recommend using [Vite](https://vitejs.dev/guide/) as your build tool. This documentation is tailored to such projects. If you're working on an older project that relies on Rollup, please refer to the section further below.
```
Comment (laurakalbag) [inline docs/recipes/svelte.md:3]:
```suggestion
This guides you through setting up XState in a Svelte project. For new Svelte or SvelteKit projects, we recommend using [Vite](https://vitejs.dev/guide/) as your build tool, and we’ve tailored this documentation to such projects. If you're working on an older project that relies on Rollup, please refer to the [Legacy Svelte projects based on Rollup section](#legacy-svelte-projects-based-on-rollup) below.
```

File: docs/recipes/svelte.md [hunk 3]
```diff
@@ -1,8 +1,15 @@
...
 
-The only gotcha is that you'll need this extra package.
+## Svelte projects based on Vite
+
+XState integrates great with Svelte & SvelteKit, and especially [Svelte stores!](https://svelte.dev/docs#svelte_store). The official `@xstate/svelte` package is recommended to get the most out of XState and Svelte.
```
Comment (laurakalbag) [inline docs/recipes/svelte.md:7]:
```suggestion
XState integrates great with Svelte & SvelteKit, and especially [Svelte stores](https://svelte.dev/docs#svelte_store). We recommend the official `@xstate/svelte` package to get the most out of XState and Svelte.
```

File: docs/recipes/svelte.md [hunk 4]
```diff
@@ -1,8 +1,15 @@
...
+
+- Details about the installation and usage example can be found in the new docs on the page about the [xstate svelte package](https://stately.ai/docs/xstate/packages/xstate-svelte).
```
Comment (laurakalbag) [inline docs/recipes/svelte.md:9]:
```suggestion
- Find installation details and example usage in the new docs on the [xstate svelte package](https://stately.ai/docs/xstate/packages/xstate-svelte).
```

File: docs/recipes/svelte.md [hunk 5]
```diff
@@ -1,8 +1,15 @@
...
+- You can also check out a [template](https://stately.ai/docs/xstate/templates) for quickly gettings started with a minimal project.
```
Comment (laurakalbag) [inline docs/recipes/svelte.md:10]:
```suggestion
- Check out a [template](https://stately.ai/docs/xstate/templates) to get started quickly with a minimal project.
```

File: docs/recipes/svelte.md [hunk 6]
```diff
@@ -1,8 +1,15 @@
...
+
+## Legacy: Svelte projects based on Rollup
```
Comment (laurakalbag) [inline docs/recipes/svelte.md:12]:
```suggestion
## Legacy Svelte projects based on Rollup
```

**Review by davidkpiano — APPROVED** [review]

**Review by laurakalbag — CHANGES_REQUESTED** [review]
@TobiObeck thank you so much for this! We really appreciate it. I’ve made a few suggested changes just for the text because I’m an annoying nerd.

**Comment by TobiObeck** [comment]
Hey @laurakalbag , thanks for your suggestions. I like your briefer formulations and committed them all. There are only two sections where I think a bit longer version sounds more natural to me or maybe flows a bit better. But I don't have a strong opinion on it. So, leaving everything the way it is, would also be fine.

```
- This guides you through
+ This section guides you through
```

```
- Find installation details...
+ You can find installation details
```

**Review by laurakalbag — APPROVED** [review]
Thank you so much, @TobiObeck!

**Pull request merged**